x-pack/plugins/security_solution/public/timelines/components/timeline/query_bar/index.tsx

```diff
@@ -47,6 +48,36 @@ export interface QueryBarTimelineComponentProps {
   updateReduxTime: DispatchUpdateReduxTime;
 }
 
+const SearchBarContainer = styled.div`
```

Comment:
👌🏾 I like it, but let's run it by paul to make sure he's cool with this

Reply:
Sure.. but since date in timeline is absolute, the space for query bar can be constraint, that is why I went with this approach. Additionally, it looked better.